docs/source/cuda/cuda_array_interface.rst

```diff
@@ -77,7 +77,101 @@ The following are optional entries:
               to a GPU function.
 
 
-
+Synchronization
+---------------
+
+For its implicit synchronization, the consumer is expected to operate on the
+data on the default stream, or synchronized with an event on the default stream.
```

Comment:
I think it might be necessary to clarify the "consumer" as the "user" in order to accommodate situations like [in mpi4py](https://github.com/numba/numba/pull/5162#issuecomment-579896688) or [in RMM](https://github.com/numba/numba/pull/5162#issuecomment-579904027)?